src/main/resources/db/migration/V4_1__add_timestamps_dialogmeldingopplysninger.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE DIALOGMELDINGOPPLYSNINGER
+ADD COLUMN invalid BOOLEAN NOT NULL DEFAULT FALSE,
+ADD COLUMN emottak-subscription TIMESTAMP,
+ADD COLUMN journalforing TIMESTAMP,
+ADD COLUMN kafka TIMESTAMP,
```

Comment:
De andre kan fungere, men Kafka synes jeg blir et for generisk navn her.

Reply:
Det er sant, skal bytte det ut med noe annet.